oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/BanyanDB.java

```diff
@@ -131,4 +131,22 @@ enum IndexType {
     @interface TimestampColumn {
         String value();
     }
+
+    /**
+     * MeasureField defines a column as a measure's field.
```

Comment:
I think we need to explain if it is not annotated, what a column would be processed.
Such as
1. storageOnly=true without annotated
2. indexOnly=true without annotated
3. storageOnly=true with annotated
4. indexOnly=true with annotated
5. storageOnly=false and indexOnly=false with annotated
5. indexOnly=false and indexOnly=false without annotated

Reply:
I will update the comments.

Reply:
Add more details to comments.